docs/source/schema/data-model.rst

```diff
@@ -0,0 +1,100 @@
+Data Model
+===========
+
+The latest version of Augur includes a schema_ that brings together data from : 
+
+1. git repositories, including GitHub
+2. issue trackers
+3. mailing lists
+4. library dependency trees
+5. the Linux Foundation's badging program
+6. code complexity and contribution counting ... and MUCH MUCH MORE. 
+
+This document details how to create the schema as well as some information on its contents and design.
+
+-----------------------
+Creating the schema
+-----------------------
+
+This section assumes you have already installed or have access to an
+installation of PostgreSQL 10 or 11. More information on PostgreSQL can
+be found `here <https://www.postgresql.org/>`__.
+
+From your local terminal, run the following commands, where ``<host>`` the server where your PostgreSQL installation is running, ``<port>`` is the port on which your installation is exposed, and ``<password>`` is a password of your choice.
+
+It's recommended you use the default database name ``augur`` and the default user ``augur`` as shown below, but they are not required.
+
+Make sure to also replace the ``/path/to/augur/`` with the path to the root ``augur/`` directory from your current directory; if you're in the the root ``augur/`` directory, then ``schema/create_schema.sql`` should work just fine. 
+
+.. code:: bash
+
+    psql -h <host> -p <port> -c "CREATE DATABASE augur;"
+    psql -h <host> -p <port> -c "CREATE USER augur WITH ENCRYPTED PASSWORD '<password>';"
+    psql -h <host> -p <port> -c "ALTER DATABASE augur OWNER TO augur;"
+    psql -h <host> -p <port> -c "GRANT ALL PRIVILEGES ON DATABASE augur TO augur;"
+    psql -h <host> -d augur -U augur -p <port> -a -w -f /path/to/augur/schema/create_schema.sql
+
+Example (note this will **not**) work\:
+
+.. code:: bash
+
+    psql -h example.com -p 5432 -c "CREATE DATABASE augur;"
+    psql -h example.com -p 5432 -c "CREATE USER augur WITH ENCRYPTED PASSWORD 'password123';"
+    psql -h example.com -p 5432 -c "ALTER DATABASE augur OWNER TO augur;"
+    psql -h example.com -p 5432 -c "GRANT ALL PRIVILEGES ON DATABASE augur TO augur;"
+    psql -h example.com -d augur -U augur -p 5432 -a -w -f /Users/carterlandis/augur/schema/create_schema.sql
+
```

Comment:
You can go ahead and remove this section and then just add a brief point about following the instructions in the getting started for setting a database up. This way we'll only have one place to worry about when changing the documentation for how the schema is created. The link reference should be this: `../getting-started/database.html`